napari/_qt/menus/window_menu.py

```diff
@@ -1,14 +1,15 @@
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, List
 
 from napari._qt.menus._util import NapariMenu, populate_menu
 from napari.utils.translations import trans
 
 if TYPE_CHECKING:
+    from napari._qt.menus._util import MenuItem
     from napari._qt.qt_main_window import Window
 
 
 class WindowMenu(NapariMenu):
     def __init__(self, window: 'Window') -> None:
         super().__init__(trans._('&Window'), window._qt_window)
-        ACTIONS = []
+        ACTIONS: List[MenuItem] = []
```

Comment:
@Czaki is the plan to drop support for 3.8 in version 0.5.0? If so maybe change to `list` here.